esrally/rally.py

```diff
@@ -167,6 +167,10 @@ def positive_number(v):
         track_source_group.add_argument(
             "--track-path",
             help="Define the path to a track.")
+        track_source_group.add_argument(
+            "--track-revision",
+            help="Define a specific revision in the tracks repository that Rally should use.",
```

Comment:
nit: `track repository`

Reply:
@danielmitterdorfer Thank you for the review. I addressed you comments. Thanks!